pulsar/consumer.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // Pair of a Consumer and Message
 type ConsumerMessage struct {
-	Consumer
```

Comment:
this changes the user API. having a reference to the `Consumer` object is to give the chance to the user to call `consumer.Ack()` with the message. If we pass an `interface{}` it would have to downcast

Reply:
what other solution do you propose to fix this bug?